tmt/steps/provision/bootc.py

```diff
@@ -0,0 +1,232 @@
+import dataclasses
+import uuid
+from string import Template
+from typing import Optional
+
+import tmt
+import tmt.base
+import tmt.log
+import tmt.steps
+import tmt.steps.provision
+import tmt.steps.provision.testcloud
+import tmt.utils
+from tmt.steps.provision.testcloud import GuestTestcloud
+from tmt.utils import field
+
+DEFAULT_IMAGE_BUILDER = "quay.io/centos-bootc/bootc-image-builder:latest"
+
+
+class GuestBootc(GuestTestcloud):
+    containerimage: str
+
+    def __init__(self,
+                 *,
+                 data: tmt.steps.provision.GuestData,
+                 name: Optional[str] = None,
+                 parent: Optional[tmt.utils.Common] = None,
+                 logger: tmt.log.Logger,
+                 containerimage: Optional[str]) -> None:
+        super().__init__(data=data, logger=logger, parent=parent, name=name)
+
+        if containerimage:
+            self.containerimage = containerimage
+
+    def remove(self) -> None:
+        if self.containerimage:
```

Comment:
I'm not 100 % sure, but if `self.containerimage` was not assigned a value in `__init__()`, this test might fail because Python would report a missing attribute. It would be better to mark `containerimage` as `Optional[str]` and give it `None` as a default - it obviously can be irrelevant, `None` is easier for linters to track and prevent the use.